src/integrators/ScaleIntegrator.C

```diff
@@ -0,0 +1,119 @@
+#include "ScaleIntegrator.h"
+
+namespace platypus
+{
+
+void
+ScaleIntegrator::SetIntRule(const mfem::IntegrationRule * ir)
```

Comment:
For test coverage purposes, this method should be covered in `TestScaleIntegrator`